pyproject.toml

```diff
@@ -32,15 +32,14 @@ classifiers = [
     'Programming Language :: Python :: 3.8',
     'Programming Language :: Python :: 3.9',
     'Programming Language :: Python :: 3.10',
-    'Programming Language :: Python :: 3.11',
```

Comment:
Can you remove the changes made to `pyproject.toml` and `poetry.lock`?

Reply:
Reverted to versions currently in master branch